app/assets/stylesheets/application/_variables.scss

```diff
@@ -1,44 +1,47 @@
-// Custom variables
-$red: #cc0000;
-$darkred: #950002;
-$darkgrey: #353535;
-$lightgrey: #ddd;
-$mediumgrey: #636363;
+$base-font-size:   16px;
+$small-font-size:  $base-font-size * 0.875;
+$base-line-height: 1.5;
+
+/* colors */
+$eff-red: #ec1e1e;
+$red: #FF0000;
+$dark-red: #b80101;
+
 $blue: #0f6499;
 $green: #7dcc00;
-$sans-serif: 'Roboto';
+$off-white: #EAEAEA;
```

Comment:
Would it be a lot of work at this point to add semantic names to the theme colors and use those in most instances, rather than the color name directly? Something like

```scss
// _variables.scss
$off-white: #EAEAEA;
$gray-dark: #999999; // footer text
$charcoal: #2D2D2D;

// _theme.scss
$page-background: $charcoal;
$tool-background: $gray-dark;
$text-color: $off-white;
$header-color: white;

// _base.scss
body {
  background: $page-background;
  color: $text-color;
}

// _action_page.scss
#tools .tool {
  background: $tool-background;

  .tool-title {
    color: $header-color;
  }
}
```

Reply:
We could. I guess I've always preferred just name the colors.  Perhaps we could name the variables by color, and then have them extended for semantic purposes? I've seen a lot of sites where a color variable will be something like "page-background" but then gets used for other purposes, and it gets weird.